.circleci/config.yml

```diff
@@ -170,6 +170,22 @@ jobs:
           command: docker run app:build make lint
 
 
+  build-job-quicksuggest-2-bq:
```

Comment:
My personal preference would be to avoid punctuation and call this job `quicksuggest2bq`. We already have two jobs in this repo that follow that convention. It may make some people's eyes twitch, though 👁️

Reply:
@jklukas would you rather have the whole package renamed (both the directory and the files) or just this one job name?

Reply:
Sorry for not being clear there. I would want all the directories and files to consistently use `quicksuggest2bq`